src/frontend/qt_sdl/EmuInstance.h

```diff
@@ -279,6 +281,9 @@ class EmuInstance
     bool fastForwardToggled;
     bool slowmoToggled;
     bool doAudioSync;
+
+    melonDS::u32 getInputMask(){return inputMask;}
+    std::map<melonDS::u8,bool> KeyboardMask; //For Lua Scripts
```

Comment:
if this is really always just 256 keys, please use std::bitset or our (better) NonStupidBitfield. Also please avoid always putting 256 into the code.